code/pizzabot.js

```diff
@@ -1,7 +1,266 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
+const vegetarian = "vegetarian";
+const hawaiian = "hawaiian";
+const pepperoni = "pepperoni";
+const capricciosa = "capricciosa";
+const margherita = "margherita";
+const pizzaPrice = 80;
 
-const pizzaPrice = 80
+/* FUNCTIONS */
 
-//Put your Javscript code here:
+/* Calculate the cost of order */
+const calculateTotalCost = (orderQuantity, pizzaPrice) => {
+  const orderTotal = orderQuantity * pizzaPrice;
+  return orderTotal;
+};
+
+/* Calculate cooking time */
+const calculateCookingTime = (orderQuantity) => {
+  let cookingTime;
+  if (orderQuantity >= 1 && orderQuantity <= 2) {
+    cookingTime = 10;
+    console.log(typeof cookingTime);
+  } else if (orderQuantity >= 3 && orderQuantity <= 5) {
+    cookingTime = 15;
+  } else if (orderQuantity >= 6) {
+    cookingTime = 20;
+  } else {
+    cookingTime = 0;
+  }
+  return cookingTime;
+};
+
+/* Place order. This function is called when the user presses the "Place Order Button." */
+const placeOrder = () => {
+  const name = capitalizeFirstLetter(document.getElementById("name").value);
+  const orderName = document.getElementById("pizza").value;
+  const orderQuantity = document.getElementById("pizza-slider").value;
+  const orderTotal = calculateTotalCost(orderQuantity, pizzaPrice);
+  let cookingTime = calculateCookingTime(orderQuantity);
+  console.log(typeof cookingTime + " cookingTime");
```

Comment:
Before pushing your code, always delete all `console.log()` occurrences, it's considered bad practise to include them on github repo